test/e2e/vsphere_workload_api_flux_test.go

```diff
@@ -0,0 +1,228 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func TestVSphereMulticlusterWorkloadClusterAPIGitHubFlux(t *testing.T) {
+	vsphere := framework.NewVSphere(t)
+	managementCluster := framework.NewClusterE2ETest(
```

Comment:
Can you remind me what name is used here if we are specifying new names for the workload clusters below? Should we maybe reuse the name and append something so that we can associate the clusters better?

Reply:
`NewWorkloadClusterName` returns a new unique name for a workload cluster based on the management cluster name. So, I think this already does what you're saying.